sites/website/src/prepare-docs.js

```diff
@@ -344,8 +357,56 @@ async function buildAPIMarkdown() {
     }
 }
 
+async function copyImages() {
+    const images = [
+        {
+            src: path.resolve(root, "examples/site-rebrand-tutorial/website.png"),
+            dest: path.resolve(
+                staticOutputDir,
+                "examples/site-rebrand-tutorial/website.png"
+            ),
+        },
+        {
+            src: path.resolve(root, "examples/site-rebrand-tutorial/site-structure.png"),
+            dest: path.resolve(
+                staticOutputDir,
+                "examples/site-rebrand-tutorial/site-structure.png"
+            ),
+        },
+        {
+            src: path.resolve(
+                root,
+                "examples/site-rebrand-tutorial/example-controls.png"
+            ),
+            dest: path.resolve(
+                staticOutputDir,
+                "examples/site-rebrand-tutorial/example-controls.png"
+            ),
+        },
+        {
+            src: path.resolve(root, "examples/site-rebrand-tutorial/side-by-side.png"),
+            dest: path.resolve(
+                staticOutputDir,
+                "examples/site-rebrand-tutorial/side-by-side.png"
+            ),
+        },
+        {
+            src: path.resolve(root, "examples/site-rebrand-tutorial/design-panel.png"),
+            dest: path.resolve(
+                staticOutputDir,
+                "examples/site-rebrand-tutorial/design-panel.png"
+            ),
+        },
+    ];
+
+    for (const img of images) {
+        await safeCopy(img.src, img.dest);
+    }
+}
+
 async function main() {
```

Comment:
Does resolution order matter here? If not, I think `return Promise.all([copyArticleMarkdown(), copyImages(), buildAPIMarkdown()])` will do this work a bit faster.

Reply:
Good call. Will fix this up.